docs/developing.asciidoc

```diff
@@ -0,0 +1,80 @@
+// begin header
+// end header
+
+toc::[]
+
+== Setting up Certman-operator for local dev
+=== Requirements
+To develop certman you will need the following:
+
+. Access to the hive repo
+. Let’s Encrypt account id and private key.
+. AWS keys and secret keys for the account you’re using.
+. Am openshift test cluster
```

Comment:
s/Am/An/

Reply:
Accepted all feedback and made changes in a new commit.